collect-user-data

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
```

Comment:
This doesn't need to be `ash` compatible.  I'll remove collect-user-data in favor of BulkEditingCookbook.md.